php/Terminus/Models/Workflow.php

```diff
@@ -81,6 +81,22 @@ public function isSuccessful() {
   }
 
   /**
+   * Detects if the workflow has Operation Logs
+   *
+   * @return [boolean] $has_logs True if worklow has logs
+   */
+  public function hasLogs() {
+    $operations = $this->operations();
+    $has_logs = false;
+    foreach ($operations as $operation) {
+      if ((boolean)$operation->get('log_output')) {
+        $has_logs = true;
+      }
```

Comment:
Ah, you responded after I'd reconsidered my comments. Currently the only idea I have to improve it is to replace `$has_logs = true;` with `return true;` and put a `return false;` at the bottom, eliminating the `$has_logs` var entirely.